index.js

```diff
@@ -85,6 +101,13 @@ export default class MapboxAccessibility {
     }
   };
 
+  _geolocate = (event) => {
+    // A hack because we can't query the geolocate control directly.
+    if (event.geolocateSource) {
+      this.geolocatePosition = this.map.getCenter();
```

Comment:
by the way I don't think this will work when the location updates and the camera doesn't follow, but I think it's fine to omit that for a first pass.

Reply:
Ah I see `trackUserLocation` moves camera automatically, I didn't see that in my copy/paste (oops). I agree though these changes get us most of the way there. Then we'll need to make a PR to mapbox-gl-js to get a reference to the control so we can listen on the correct event (`geolocate`)